### PR TITLE
Adding connection form to walkthrough

### PR DESCRIPTION
**Pull request by Cerchie**
## Summary of Changes

Updates the walkthrough with a shoutout to the new connection form 🔉 

Feedback on the copy in particular is appreciated 👍🏻 

Screenshot 2024-12-11 at 2 29 41 PM

-

## Pull request checklist

Please check if your PR fulfills the following (if applicable):

##### Other

- [x] Have you validated this change locally by [packaging](https://github.com/confluentinc/vscode/blob/main/README.md#packaging-steps) and installing the extension `.vsix` file?
  ```shell
  gulp clicktest
  ```

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -986,6 +986,16 @@
...
+          {
+            "id": "create-connections",
+            "title": "Create a New Connection",
+            "description": "Click the + icon next to Resources -> Other to establish a connection to an Apache Kafka® Cluster or Schema Registry: in Confluent Cloud, Confluent Platform, or hosted elsewhere.",
+            "media": {
+              "image": "resources/walkthrough/direct-connection-form.png",
+              "altText": "Create a new connection to a Kafka cluster or Schema Registry"
+            },
+            "completionEvents": []
+          },
```
Comment (shouples) [inline package.json:998]:
```suggestion
          {
            "id": "create-connections",
            "title": "Connect Directly to Kafka / Schema Registry",
            "description": "In the Resources view, click the $(plus) icon next to the 'Other' item to connect directly to a Kafka cluster or Schema Registry instance.\n[Open Connection Form](command:confluent.connections.direct)",
            "media": {
              "image": "resources/walkthrough/direct-connection-form.png",
              "altText": "Create a new connection to a Kafka cluster or Schema Registry"
            },
            "completionEvents": [
              "onCommand:confluent.connections.direct"
            ]
          },
```
What do you think about something like this? Having the quick-access command buttons is handy, plus we can use the theme icon ids directly to match what we display in the sidebar views/items.
image

**Comment by tikimims** [comment]
@JimGalasyn You suggestion looks good. It's concise and clear and follows better UX writing principles. How about also with a few minor nits:

"To connect to an Apache Kafka or Schema Registry cluster, click + next to Resource > Other. The cluster can be in Confluent Cloud, Confluent Platform, or hosted elsewhere." cc @shouples

**Comment by tikimims** [comment]
@shouples Are we doing sentence case in headings, see [Titles and headings section](https://confluentinc.atlassian.net/wiki/spaces/DOC/pages/3785327099/Style+Guide+for+Confluent+UI+Content#Titles-and-headings)  and section on [capitalization](https://confluentinc.atlassian.net/wiki/spaces/DOC/pages/3785327099/Style+Guide+for+Confluent+UI+Content#Capitalization).

Also noticed the orphan/widow created here in the subheading with "Confluent Cloud"
Screenshot 2024-12-12 at 10 12 28 AM (orphans/widows are single words or short lines at the beginning or end of a paragraph that appear alone on a page or column; they can disrupt flow of reading...) . See more in our [UX Content Style Guide](https://confluentinc.atlassian.net/wiki/spaces/DOC/pages/3785327099/Style+Guide+for+Confluent+UI+Content#Widows-and-orphans)

cc @JimGalasyn

**Comment by shouples** [comment]
@tikimims thanks for the feedback! Changing to sentence case makes sense. We can do another quick follow-on PR to update some of the other items in the walkthrough, but for this PR specifically it sounds like we should change `Connect Directly to Kafka / Schema Registry` to `Connect directly to Kafka / Schema Registry`?

> Also, noticed the orphan/widow created here in the subheading with "Confluent Cloud"
Unfortunately, we don't have any control over the text layout/spacing here with the `walkthroughs` -- we just have to provide the content and VS Code's UI handles the rest.

https://github.com/user-attachments/assets/40905e5c-dc41-465d-acb5-ba3593e1c373

**Review by JimGalasyn — APPROVED** [review]

**Review by shouples — APPROVED** [review]

**Pull request merged**